website/docs/tutorial-nlp.md

```diff
@@ -0,0 +1,174 @@
+---
+id: tutorial-nlp
+title: Tutorial NLP
+---
+
+# Texthero's NLP Module
+
+Texthero's NLP module features many common _Natural Language Processing_ functions applied to Pandas Series. You can see all functions with a detailed description and examples [here](https://texthero.org/docs/api-nlp). In this tutorial, we'll have a quick look at some of the functions and apply them to a real dataset.
+
+## Load Data and Preprocess
+Let's begin by loading an interesting dataset and having a first look.
+```python
+>>> import texthero as hero
+>>> import pandas as pd
+>>> df = pd.read_csv("https://raw.githubusercontent.com/jbesomi/texthero/master/dataset/superheroes_nlp_dataset.csv")
+>>> # We only keep a few interesting columns.
+>>> df = df[["name", "history_text", "powers_text"]]
+>>> df.head(3)
+```
+![](/img/tutorial-nlp-S1.png)
+
+As you can see, we are working with a dataset that's about superheroes! It features each hero's name, a texth about their history, and a text describing their superpowers. Of course, all of these can be missing (i.e. "NaN"). We will now try to generate some insights with each of the Texthero NLP functions.
+
+## Count Sentences - Who is the most well-known superhero?
+First of all, we want to know which superhero is the most important. We use the naive approach of counting the number of sentences in their history. The idea is that more well-known superheroes have a richer backstory and writers put more effort into their history.
+
+Of course, we use Texthero's `count_sentences` function.
```

Comment:
The "Of course" is disrupting the reading flow, maybe something like Textheros 's `count_sentences` function is perfectly suitable for this task, or similar

Reply:
will change :2nd_place_medal: